packages/block-library/src/latest-posts/index.php

```diff
@@ -13,7 +13,7 @@
  * @return string Returns the post content with latest posts added.
  */
 function render_block_core_latest_posts( $attributes ) {
-	$recent_posts = wp_get_recent_posts(
+	$recent_posts_query = new WP_Query(
```

Comment:
If you just loop through the posts anyway, I'd just use `get_posts()`. Make sure to pass `'suppress_filters' => false` as query arg though.

Reply:
done, Thanks